lib/syskit/runtime/remote/server/ftp_upload.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
```

Comment:
What is this folder ?

I would put it in `roby_app/log_transfer` to be honest. And move SpawnServer here (renaming it into just Server)